main.go

```diff
@@ -1,10 +1,14 @@
 package main
 
-import "github.com/data-preservation-programs/spade-tenant/db"
+import (
+	"github.com/data-preservation-programs/spade-tenant/api/v1"
+	"github.com/data-preservation-programs/spade-tenant/config"
+	"github.com/labstack/echo/v4"
+)
 
 func main() {
-	dbDsn := "postgres://postgres:password@localhost:5432/spade-tenant"
-	debug := true
-
-	db.OpenDatabase(dbDsn, debug)
```

Comment:
since you removed this, where does the DB get setup now? or will this be in a separate PR?